tests/test_cli.py

```diff
@@ -564,6 +564,42 @@ def mock_read(*_, **__):
     assert '{"foo": "bar"}' in result.output
 
 
+def test_cli_read_command_with_chunk_size(fs, monkeypatch):
+    """Test ralph read command with a chunk_size option."""
+
+    def get_mock_read_bytes(expected_chunk_size: int):
+        """Return a `_read_bytes` function checking the `chunk_size` option."""
+
+        def mock_read_bytes(self, query, target, chunk_size, *_):
+            """Check chunk_size and return an archive."""
+            assert chunk_size == expected_chunk_size
+            yield b'{"foo": "bar"}'
```

Comment:
Shouldn't this be `yield b'{"foo": "bar"}'[:chunk_size]` , or am I missing something ?

Reply:
You are right) That would be more rigorous) Updated the test)